src/jit/jitconfigvalues.h

```diff
@@ -206,6 +205,10 @@ CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 0)
 CONFIG_INTEGER(JitEnableNoWayAssert, W("JitEnableNoWayAssert"), 1)
 #endif // !defined(DEBUG) && !defined(_DEBUG)
 
+//#if MEASURE_MEM_ALLOC -- macro not always available here due to #include ordering
```

Comment:
It would be nice, somewhere, to have a comment that describes the issue (i.e. AFAICT 1) that the config value is defined in a clr-wide include file that doesn't include jit.h, and 2) possible other ordering issues), with a TODO-Cleanup that suggests perhaps this could be addressed.

---
In reply to: [78436756](https://github.com/dotnet/coreclr/pull/7119#discussion_r78436756) [](ancestors = 78436756)